alignment/sequencealigner_test.py

```diff
@@ -0,0 +1,254 @@
+"""
```

Comment:
I don't mind so much. But maybe we could get the other code into the right shape. It was still work in progress.